test/test.js

```diff
@@ -369,4 +327,58 @@ describe('YeldaChat', () => {
       })
     })
   })
+
+  /**
+   * Since we are resetting and updating the mockRequest if platformSettings test cases run with
+   * other test cases parallely it causes error due to asynchronous nature.
+   * So "after" root level hook used to run the platformSettings test cases after
+   * all other test cases are finished
+   */
```

Comment:
i am sorry, but in term of pure English, it's not clear
Could you try to rephrase it, and maybe use some ponctuation like coma "," a bit more ?